examples/00_quick_start/sequential_recsys_amazondataset.ipynb

```diff
@@ -469,30 +525,257 @@
+    }
+   ],
+   "source": [
+    "model_best_trained.predict(test_file, output_file)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Exciting. Now let's start our quick journey of online serving. \n",
     "\n",
-    "\\[3\\] Tang, Jiaxi, and Ke Wang. Personalized top-n sequential recommendation via convolutional sequence embedding. Proceedings of the Eleventh ACM International Conference on Web Search and Data Mining. ACM, 2018.\n",
+    "For efficient and flexible serving, usually we only keep the necessary computation nodes and froze the TF model to a single pb file, so that we can easily compute scores with this unified pb file in both Python or Java:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 16,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "WARNING:tensorflow:From <ipython-input-16-dd62857b20ba>:6: convert_variables_to_constants (from tensorflow.python.framework.graph_util_impl) is deprecated and will be removed in a future version.\n",
+      "Instructions for updating:\n",
+      "Use `tf.compat.v1.graph_util.convert_variables_to_constants`\n",
+      "WARNING:tensorflow:From /home/v-xdeng/.conda/envs/reco_tf15/lib/python3.6/site-packages/tensorflow_core/python/framework/graph_util_impl.py:277: extract_sub_graph (from tensorflow.python.framework.graph_util_impl) is deprecated and will be removed in a future version.\n",
+      "Instructions for updating:\n",
+      "Use `tf.compat.v1.graph_util.extract_sub_graph`\n",
+      "INFO:tensorflow:Froze 61 variables.\n",
+      "INFO:tensorflow:Converted 61 variables to const ops.\n"
+     ]
+    }
+   ],
+   "source": [
     "\n",
-    "\\[4\\] Yuan, F., Karatzoglou, A., Arapakis, I., Jose, J. M., & He, X. A Simple Convolutional Generative Network for Next Item Recommendation. WSDM, 2019"
+    "with model_best_trained.sess as sess:\n",
+    "    graph_def = model_best_trained.graph.as_graph_def()\n",
+    "    output_graph_def = tf.graph_util.convert_variables_to_constants(\n",
+    "        sess,\n",
+    "        graph_def,\n",
+    "        [\"pred\"]\n",
+    "    )\n",
+    "\n",
+    "    outfilepath = os.path.join(hparams.MODEL_DIR, \"serving_model.pb\")\n",
+    "    with tf.gfile.GFile(outfilepath, 'wb') as f:\n",
+    "        f.write(output_graph_def.SerializeToString())\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The serving logic is as simple as feeding the feature values to the corresponding input nodes, and fetch the score from the output node. \n",
+    "\n",
+    "In our model, input nodes are some placeholders and control variables (such as is_training, layer_keeps). We can get the nodes by their name:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 17,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "class LoadFrozedPredModel:\n",
```

Comment:
Can this-part ("froze TF model to a single pb file and load it") be moved into utils? I personally love to see this kinds of details, but could be somewhat hardcore for quickstart :stuck_out_tongue_winking_eye:

Reply:
work in progress XD